bitnami/postgresql/templates/statefulset-slaves.yaml

```diff
@@ -286,6 +286,8 @@ spec:
             timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
             successThreshold: {{ .Values.livenessProbe.successThreshold }}
             failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+          {{- else if .Values.customLivenessProbe }}
```

Comment:
I think this logic is confusing.
To use the customLiveness Probes the user will need to set `livenessProbe.enabled=false` and then `customLivenessProbe`. I would do it like:
```
{{- if and .Values.livenessProbe.enabled .Values.customLivenessProbe }}
...
{{- else if .Values.livenessProbe.enabled  }}
...
{{- else }}
...
{{- end }}
```

And something similar for the readiness probe

Reply:
It's already set like that in most of the charts. We have similar logics in other parameters like `externalDatabase`. For consistency I would leave it like the rest.

Reply:
But I would add it as something to review in the future

Reply:
👍